### DISCUSSION TITLE
Dịch `Instance-based learning`

### DISCUSSION THREAD

**Discussion by cuongvng** [mlbvn/handson-ml2-vn#13] [Terminology] [answered]
Hiện tại mình đang dịch từ này là `học dựa trên thực thể`. Mọi người góp ý thêm nhé @mlbvn/handson-ml2

**Comment by ngcthuong** [comment]
Tên khác của nó là memory-based learning, dùng từ gì được nhỉ?
- Học thuộc lòng --> không hẳn, vì gần nghĩa học vẹt, chỉ nhớ mẫu sẵn có
- Học xuông? Học chay? -- học không có thực hành, xem chừng cũng không phải. 
- Học khuôn? -- học theo các các khuôn mẫu sẵn có, ít linh hoạt
- Học gạo, học tủ? học ko biết áp dụng, Cũng không phải.

**Reply by forummlcb** [reply]
Một vote cho `học dựa trên mẫu`, `instance` ở đây cũng là `sample`.

**Reply by ngcthuong** [reply]
Cắt xuống được 3 từ khóa không nhỉ?

**Reply by quangnhat185** [reply]
`Học theo mẫu` thì sao nhỉ?

**Reply by minhduc0711** [reply]
mình nghĩ nên để là `học dựa trên mẫu` cho tương xứng với `model-based learning` - `học dựa trên mô hình`

**Comment by minhduc0711** [comment, marked as answer]
chốt là `học dựa trên mẫu` nhé mọi người
